docs/login_and_acl.md

```diff
@@ -129,7 +128,7 @@ If you are running setup for the first time, ensure you have run `azd env set AZ
 
 Ensure you run `azd env set AZURE_USE_AUTHENTICATION` to enable the login UI once you have setup the two Azure AD apps before you deploy or run the application. The login UI will not appear unless all [required environment variables](#environment-variables-reference) have been setup.
 
-In both the chat and ask a question modes, under **Developer settings** optional **Use oid security filter** and **Use groups security filter** checkboxes will appear. The oid (User ID) filter maps to the `oids` field in the search index and the groups (Group ID) filter maps to the `groups` field in the search index. Use the optional scripts included in the sample to manage values for these fields.
+In both the chat and ask a question modes, under **Developer settings** optional **Use oid security filter** and **Use groups security filter** checkboxes will appear. The oid (User ID) filter maps to the `oids` field in the search index and the groups (Group ID) filter maps to the `groups` field in the search index. Once [data with access control information](#add-access-control-data) is present, the filters will restrict results for the logged in user correctly. If `AZURE_ENFORCE_ACCESS_CONTROL` has been set, then access control information is required to get any results at all.
```

Comment:
"access control information" isn't clear to me. Do you mean that those boxes must be checked? I think we auto-check them, right, and we don't let them disable if ENFORCE_ACCESS_CONTROL is true:

```
  <Checkbox
      className={styles.chatSettingsSeparator}
      checked={useOidSecurityFilter || requireAccessControl}
      label="Use oid security filter"
      disabled={!isLoggedIn(client) || requireAccessControl}
      onChange={onUseOidSecurityFilterChange}
  />
```

Or are you implying something about the index here?

Reply:
the wording is bad here, I can be more explicit. Good callout